ground/src/main/java/com/google/android/ground/model/geometry/Geometry.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.model.geometry
+
+import com.google.common.collect.ImmutableList
+
+/** The class of geometry errors.
```

Comment:
Please run `google-java-format` on all files, I believe it will place the first phrase in multiline Javadoc on the following line.

Also, if you mean 'class' as in 'a group, set, or kind sharing common attributes' it may be better to rephrase to avoid collision with the OOP term of the same name

Reply:
Hm, when I run the formatter on kotlin files it doesn't seem to automatically handle doc comments for some reason. Good suggestion, changed the terminology.